LoginApp/app/src/main/java/com/example/loginapp/localDb/RealmUserDetailsOperation.kt

```diff
@@ -0,0 +1,42 @@
+package com.example.loginapp.localDb
+
+import com.example.loginapp.entity.UserDetails
+import io.realm.Realm
+import io.realm.RealmConfiguration
+
+object RealmUserDetailsOperation {
+
+    private val realmInstance : Realm = Realm.getInstance(RealmConfig.getRealmConfiguration() as RealmConfiguration)
+    private val EMAIL_FIELD : String = "email"
+    private val PASSWORD_FIELD : String = "password"
+
+    fun writeUserDetailsToRealm(
+        email : String,
+        password : String,
+        name : String,
+        phone : String,
+        age : String
+    ) {
+        realmInstance.executeTransaction {
+            val userDetails = it.createObject(UserDetails::class.java)
+            userDetails.apply {
+                this.email = email
+                this.password = password
+                this.name = name
+                this.phone = phone
+                this.age = age
+            }
+        }
+    }
+
+    fun readUserDetailsFromRealm(email : String, password: String) : UserDetails? {
```

Comment:
@NayakRoshan no check whether `realmInstance` is still open or not

Reply:
@rishabh-jain-1990 so I have to use getLocalInstanceCount to check right?